gui/slick/interfaces/default/inc_bottom.mako

```diff
@@ -66,11 +66,15 @@
         <!--
```

Comment:
I wondered that too.
I guess these options were here before being moved in the top menu.
I'm not sure if it is necessary to keep it.